graphrag/app/supportai/supportai.py

```diff
@@ -414,8 +436,36 @@ def create_ingest(
         data_stream_conn = data_stream_conn.replace(
             "@source_config@", json.dumps(connector)
         )
-    elif ingest_config.data_source.lower() == "local" or ingest_config.data_source.lower() == "remote":
-        pass
+    elif ingest_config.data_source.lower() == "local":
+        # Handle multi-format processing for local files
+        if ingest_config.file_format.lower() == "multi":
+            folder_path = ingest_config.data_source_config.get("folder_path", None)
+            if folder_path is None:
+                raise Exception("Folder path not provided for local multi-format processing")
+
+            try:
+                # Process local folder and extract text from all supported files
+                local_processing_result = process_local_folder(folder_path, graphname=graphname)
+                if local_processing_result.get("statusCode") != 200:
+                    raise Exception(f"Local folder processing failed: {local_processing_result}")
+
+                logger.info(f"Starting local folder text extraction and TigerGraph loading...")
+
+                processed_files = local_processing_result.get("files", [])
+                successful_files = [f for f in processed_files if f.get('status') == 'success']
+
+                # Get the JSONL file that was already created during processing
+                jsonl_filepath = local_processing_result.get("jsonl_file_path")
+                if not jsonl_filepath:
+                    raise Exception("JSONL file was not created during local folder processing")
+
+                # Store the processed files to response
+                res_ingest_config["json_filepath"] = jsonl_filepath
```

Comment:
@prinskumar-tigergraph Please review if we should pass the combined jsonl file or the converted processed_file to the response, which will be used by runDocumentIngest() later.